SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/CookAndGenerateSchemaCommandlet.cpp

```diff
@@ -68,7 +68,7 @@ UCookAndGenerateSchemaCommandlet::UCookAndGenerateSchemaCommandlet()
 int32 UCookAndGenerateSchemaCommandlet::Main(const FString& CmdLineParams)
 {
 	// Force spatial networking
-	GetMutableDefault<UGeneralProjectSettings>()->bSpatialNetworking = true;
+	GetMutableDefault<UGeneralProjectSettings>()->SetUseSpatialNetworking(true);
```

Comment:
Driveby, could you wrap this in the #if ENGINE_MINOR_VERSION tags, like above?
cc: @mironec

Reply:
Ah yes, I must have missed this, it should be wrapped in #if ENGINE_MINOR_VERSION <= 22
since this is no longer required in 4.23.